pkgs/development/tools/tracy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig}:
+stdenv.mkDerivation rec {
+  name    = "tracy-${version}";
+  version = "0.6.3";
+
+  buildInputs = [tbb gtk2 glfw pkgconfig];
```

Comment:
Why is `freetype` missing from here?
Also, I suppose that `pkgconfig` should go to `nativeBuildInputs`.

Reply:
Also, for macos you need to add `Carbon AppKit` from `pkgs.darwing.apple_sdk.frameworks`.
I've seen somewhere snippets for conditional adding of these deps...

`gtk`, on the other hand, is not needed on macOS.